src/helpers.ts

```diff
@@ -36,6 +36,20 @@ export const EMPTY_HASH_256 = '0x00000000000000000000000000000000000000000000000
 
 export const pipelineAsync = util.promisify(stream.pipeline);
 
+// This class and enum are to throw errors that are supposed to be sent to the client
+enum RespErrorType {
+  invalid_tx_id = 'Invalid tx id',
+}
+export class RespError extends Error {
```

Comment:
This would make more sense if it was named something like `InvalidRequestError`

Reply:
I want to use the same class for all the errors that are to be sent to the client and not logged at all. So, would be using the same class for #854. That was the reason behind naming this class as `RespError`. But I'll convert it to `InvalidRequestError`, if that is the category of errors we are concerned about.